.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -34,12 +34,12 @@ body:
       label: "CPython versions tested on:"
       multiple: true
       options:
-        - "3.8"
         - "3.9"
         - "3.10"
         - "3.11"
         - "3.12"
         - "3.13"
+        - "3.14"
         - "CPython main branch"
```

Comment:
3.14 is still the `main` branch until May, but people might have installed 3.14 alpha via the installer.

How did we do this during 3.13 alpha?

Reply:
3.13 was added here right after 3.13a1 was released: #111083